README.md

```diff
@@ -23,6 +23,13 @@ should be inside an ETS key for this to work (or a pid).
 When some node is down, we remove all records that are owned by this node.
 When a node reappears, the records are added back.
 
+You can also use totally random keys. In this case all keys should be unique.
+We also support tables with type=bag. In such case all records would remain in the table.
+
+You can provide the `handle_conflict` function, which would be called if two records have
```

Comment:
Is there a any information what happens when there is no `handle_conflict` function?

Reply:
only in tests, could write though.
Basically, if we have node A and B, data on B would be overwritten with data from A and vice versa.